linux_os/guide/services/ssh/ssh_server/sshd_set_maxstartups/rule.yml

```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+title: 'Ensure SSH MaxStartups is configured'
+
+description: |-
+    To protect a system from denial of service due to a large number of pending
+    authentication connection attempts, use the rate limiting function of MaxStartups
+    to protect availability of sshd logins and prevent overwhelming the daemon.
+
+rationale: |-
+    The MaxStartups parameter specifies the maximum number of concurrent
+    unauthenticated connections to the SSH daemon. Additional connections will be
+    dropped until authentication succeeds or the LoginGraceTime expires for a
+    connection. The sshd default is 10. By default, this rule uses the
+    CIS-recommended value of 'MaxStartups 10:30:60', which adds 'random early drop'
+    functionality.
```

Comment:
What value is expected for the parameter `MaxStartups` depends more on the variable than the rule itself.
Here you can mention the variable using following syntax: `{{{ xccdf_value("var_sshd_set_maxstartups") }}}`.
Example: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml#L12

This will render the rule nicely in the reports and guides with the actual value selected for the profile.
Also, I think this should be part of the description, rather then rationale.

Reply:
This makes sense. I also think you're right and maybe the rationale and description fields I'd settled on here possibly work better the other way around, so I've swapped things a bit.